src/Routing/Router.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Umpirsky\I18nRoutingBundle\Routing;
+
+use Symfony\Component\Routing\Router as SymfonyRouter;
+
+class Router extends SymfonyRouter
```

Comment:
@kbond When using `prefix_except_default` strategy, and you are on `/es/route66` url, you want all urls on the page to be `/es` prefixed.